utils/lists.ts

```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Randomized adventurer meals",
     url: "https://etherscan.io/address/0x615a610649e656485d9baf0ebe525496d7b78e24",
   },
+  {
+    name: "Cangjie(倉頡)",
+    description: "The first Chinese characters Loot project with Yinyang, Wuxing and Zhouyi ☯.",
```

Comment:
Length looks good. Please remove the trailing period in description and that should be it

Reply:
Done. Thanks for your patient.